CHANGELOG.md

```diff
@@ -108,6 +108,9 @@ release.
 - BREAKING: Rename `http.server.request.size` metric to `http.server.request.body.size`
   and `http.server.response.size` metric to `http.server.response.body.size`
   ([#247](https://github.com/open-telemetry/semantic-conventions/pull/247))
+- BREAKING: Rename `telemetry.auto.name` resource attribute `telemetry.distro.name`
+  and add `telemetry.distro.version` resource attribute
+  ([#178](https://github.com/open-telemetry/semantic-conventions/pull/178))
```

Comment:
Isn't it the other way around? `telemetry.auto.version` already exists, but not `telemetry.auto.name`.

Reply:
good catch :smile: